index.html

```diff
@@ -2566,22 +2566,21 @@ <h3>Inverse Triple Constraints</h3>
   }
 }</pre>
             <pre class="nohighlight instance turtle pass">
-	<span class="fullURL">inst:Issue1</span> <span class="keyword predicate">a</span> <span class="object"><span class="type">ex:</span><span class="constant">Issue</span></span> ;
-	<span class="state lowlight1 top bot"><span class="predicate"><span class="type">ex:</span><span class="constant">state</span></span>        <span class="object"><span class="type">ex:</span><span class="constant">unassigned</span></span></span> .</pre>
-
-<pre class="nohighlight instance turtle pass">
-	<span class="fullURL">inst:User3</span> <span class="keyword predicate">a</span> <span class="object"><span class="type">foaf:</span><span class="constant">Person</span></span> ;
+<span class="issue1 lowllight1 top"><span class="fullURL">inst:Issue1</span> <span class="keyword predicate">a</span> <span class="object"><span class="type">ex:</span><span class="constant">Issue</span></span> ;</span>
```

Comment:
"For inverse triple constraints, we have those as an integrated example
because ShEx itself allows you to get mixed results. We want to convey
the idea that you can have a block of data that gets mixed results."

A try-it button would make this clear.